helm/hpcc/templates/dali.yaml

```diff
@@ -46,12 +46,20 @@ spec:
         server: {{ $dali.name | quote }}
         app: dali
         helmVersion: 8.3.0-trunk0
+{{- if hasKey $.Values.global "metrics" }}
```

Comment:
does this and other places that reference hasKey $.Values.global, need to use the merged metrics from generateMetricsConfig too? (see other comment re. fromYaml).
If so, you could set $mymetrics once and reference elsewhere within the template.

Reply:
It doesn't appear dali nor roxie merge global.metrics into local metrics.